### PR TITLE
Update to Webpack 5.0.0-alpha.16

### PR DESCRIPTION
**Pull request by guybedford**
This updates Webpack to the latest alpha release.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -331,7 +331,7 @@ module.exports = (
     }
 
     for (const chunk of stats.compilation.chunks) {
-      finalizeOutput(chunk.files[0]);
+      finalizeOutput(chunk.files.values().next().value);
```
Comment (styfle) [inline src/index.js]: There's a merge conflict here after reverting the API in the other PR.

In fact, there is no `finalizeOutput()` function anymore 🤷‍♂

Comment (guybedford) [reply src/index.js]: Just rebased.

File: test/integration/twilio.js [hunk 2]
```diff
@@ -7,3 +7,4 @@ try {
   }
 }
 
+
```
Comment (styfle) [inline test/integration/twilio.js]: The only code change is a newline? I thought you had to change the chunkFiles?

Comment (guybedford) [reply test/integration/twilio.js]: Nicely spotted, yes that shouldn't be there.

**Pull request merged**